test/alternator/test_returnconsumedcapacity.py

```diff
@@ -126,6 +126,24 @@ def test_long_put(test_table):
     consumed_capacity = response['ConsumedCapacity']
     assert 2 == consumed_capacity["CapacityUnits"]
 
+# WCU is calculated based on 1KB block size.
+# The test validate that when calculating the WCU of a put item
+# over an existing item, the longer of the items is the
+# limit calculation
```

Comment:
Also, would be nice to have another version of the test which uses  force_read_before_write configuration option without ReturnValues.